tests/Doctrine/Tests/Persistence/ManagerRegistryTest.php

```diff
@@ -54,7 +55,11 @@ public function testGetManagerForProxyInterface()
     public function testGetManagerForInvalidClass()
     {
         $this->expectException(ReflectionException::class);
-        $this->expectExceptionMessage('Class Doctrine\Tests\Persistence\TestObjectInexistent does not exist');
+        $this->expectExceptionMessage(
+            PHP_VERSION_ID < 80000?
```

Comment:
This is not an CS violation?

Reply:
Apparently not. Note that we use `doctrine/coding-standard` 6 though.

Reply:
Space before question mark

Reply:
If that space for `?` and `:` doesn't violate the current standard.

Reply:
Done